logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -37,3 +47,59 @@ def format_stackdriver_json(record, message):
     }
 
     return json.dumps(payload)
+
+
+def get_trace_id_from_flask():
+    """Get trace_id from flask request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
+    """
+    if not flask or not flask.request:
+        return None
+
+    header = flask.request.headers.get(_FLASK_TRACE_HEADER)
+
+    if not header:
+        return None
+
+    trace_id = header.split('/')[0]
+
+    return trace_id
+
+
+def get_trace_id_from_django():
+    """Get trace_id from django request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
+    """
+    request = _get_request()
+
+    if not request:
+        return None
+
+    try:
+        header = request.META[_DJANGO_TRACE_HEADER]
+    except KeyError:
+        return None
+
+    trace_id = header.split('/')[0]
+
+    return trace_id
+
+
+def get_trace_id():
+    """Helper to get trace_id from web application request header.
+
+    :rtype: str
+    :returns: Trace_id in HTTP request headers.
+    """
+    checkers = [get_trace_id_from_django, get_trace_id_from_flask]
+
+    for checker in checkers:
+        trace_id = checker()
+        if trace_id is not None:
+            return trace_id
+
+    return trace_id
```

Comment:
Just `return None`. There's no chance of this being any other value, so an explicit `return None` conveys the real behavior here.